src/main/java/com/google/codeu/servlets/MessageFeedServlet.java

```diff
@@ -63,10 +68,16 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     String userEmail = userService.getCurrentUser().getEmail();
```

Comment:
Nit: Can you make sure that the indentation, here and everywhere, is consistent with the style guide?